benchmarks/run_stardis.py

```diff
@@ -6,7 +6,7 @@
 from astropy import units as u
 
 from tardis.io.atom_data import AtomData
-from tardis.io.config_validator import validate_yaml
+from tardis.io.configuration.config_validator import validate_yaml
 from tardis.io.config_reader import Configuration
```

Comment:
I think this guy has to be changed too

Reply:
I believe this is [fixed](https://github.com/tardis-sn/stardis/pull/142/commits/ef7c4fe17641cd8ca951ac7c748a24876a966f56)